docs/guides/useful-libraries-for-testing.md

```diff
@@ -0,0 +1,16 @@
+## Useful Libraries for Testing
+
+`vue-test-utils` provides useful methods for testing Vue components. Community members have also written some additional libraries which either extend `vue-test-utils` with extra useful methods, or provide tools for testing other things found in Vue applications.
```

Comment:
As far as I can tell VTU is usually refered as `Vue Test Utils` instead of `vue-test-utils`

```suggestion
Vue Test Utils provides useful methods for testing Vue components. Community members have also written some additional libraries which either extend Vue Test Utils with extra useful methods, or provide tools for testing other things found in Vue applications.
```

Reply:
Was not sure how to revert to VTU - thanks.

Reply:
Fixed.